cdap-api/src/main/java/co/cask/cdap/api/plugin/PluginConfig.java

```diff
@@ -34,6 +35,10 @@
   private PluginProperties properties;
   private Set<String> macroFields;
 
+  public PluginConfig() {
+    this.macroFields = Collections.emptySet();
```

Comment:
we could, that would also mean having to revisit hydrator plugins that use the check to check if it's called from a test case though